File: BUDDY/buddy.py
import requests

r = requests.get('https://valorant-api.com/v1/buddies?language=ko-KR')
kor = r.json()
b = kor['data']
all = len(b)
counts = 0
for counts in range(0, all):
    korname = b[counts]['displayName'].upper()
    img = b[counts]['displayIcon']
    print(f'{korname}\n{img}\n{counts + 1}/{all}')

File: BUDDY/buddy.js
const axios = require('axios');

axios.get('https://valorant-api.com/v1/buddies?language=ko-KR', {'User-Agent': 'info'}).then((Response)=>{
    var data = Response.data['data']
    for (var i = 0; i < data.length; i++) {
        var korname = data[i]['displayName']
        var korimg = data[i]['displayIcon']
        console.log(korname)
        console.log(korimg)
        console.log(i+1 + '/' + data.length)
    }
}).catch((Error)=>{
    console.log(Error);
}) 
